worker_binder.go

```
@@ -0,0 +1,93 @@
+
+func (qs *workerBinder[T, R]) BindWithDistributedQueue(dq IDistributedQueue) DistributedQueue[T, R] {
+	qs.worker.start()
+	defer dq.Subscribe(qs.handleQueueSubscription)
+
+	queue := NewDistributedQueue[T, R](dq)
+	qs.worker.setQueue(dq)
+	return queue
+}
+
+func (qs *workerBinder[T, R]) BindWithDistributedPriorityQueue(dq IDistributedPriorityQueue) DistributedPriorityQueue[T, R] {
+	qs.worker.start()
```

golangci-lint (1.64.8)

Error return value of `qs.worker.start` is not checked

(errcheck)